Development

SebastianPetak opened the pull request:
@KasaiMagi Making a contact management system.  It's a CRUD app that allows you to manage a contacts list.  I will probably add an account feature.  I'm making this to have more experience with database schema, user authentication, routing, and APIs.

SebastianPetak committed:
Modify contact model, edit contacts page
Contact model takes a name value, but it should be firstName and lastName.
Contacts page should display information in a more uniform manner.
Modify the contact model to take a firstName, lastName, picture.
Add css folder with styles and serve it statically.

SebastianPetak committed:
Add contact page
Create a contact page to display information about
the users contact. Create the route and add basic
styles to the page.

SebastianPetak committed:
Create contact api route
Create a route to access an individual contact by their _id.

SebastianPetak committed:
Create User model & modify Contact model

tdbishop commented on views/contacts.ejs:
```diff
@@ -5,28 +5,38 @@
-
-	</style>
 </head>
 <body>
```
is there a way to template this so both .ejs files share their code outside of `<body>`?

SebastianPetak replied:
Definitely haha. Sorry this app is a mess atm. I actually went though a whole TreeHouse course on authentication, cookies, and sessions yesterday, so I'm going to refactor what I've already written. I really dislike using the <%= syntax that ejs provides. That is really difficult to read! I wish it was `{{ code }}`. I might mess around with pug.

tdbishop commented:
nice!

SebastianPetak committed:
Create html templates, fix models
Our views have the same information on multiple pages.
Create ejs templates to eliminate redundant html.
Both models are missing commas.  Fix typos in models.
